tests/plan/client/env/optional.cue

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"dagger.io/dagger"
+	"dagger.io/dagger/core"
+)
+
+dagger.#Plan & {
+	client: env: {
+		TEST_DEFAULT:      string | *"hello world"
+		TEST_OPTIONAL?:    dagger.#Secret
+		TEST_OPT_DEFAULT?: string | *"hello dagger"
```

Comment:
Any reason you have a default on this? It will either be unset or be the string that is set in the environment. Are you testing something specific with the default?

Reply:
Just that someone might use it, not that’s a proper thing to do.

Reply:
I can either add a comment explaining or maybe it’s not useful to test.

Reply:
No longer useful, I removed that 🙂

Reply:
👍🏻 Sounds good!